test/extensions/filters/http/cdn_loop/filter_integration_test.cc

```diff
@@ -67,6 +67,24 @@ TEST_P(CdnLoopFilterIntegrationTest, CdnLoopHeaderWithOtherCdns) {
   EXPECT_EQ("200", response->headers().getStatusValue());
 }
 
+TEST_P(CdnLoopFilterIntegrationTest, MultipleCdnLoopHeaders) {
```

Comment:
Would this have failed before the inline header change?  If not maybe just remove this test and edit CdnLoop2Allowed3Seen to have multiple header lines.

Reply:
Yup, it failed before the inline header change.  It hit the `appendCopy` appends to the first header it sees problem.